<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" source-language="en" target-language="tr" original="../ExtensionResources.resx">
    <body>
      <trans-unit id="FailedToCreateRetryDirectoryBecauseOfCollision">
        <source>Failed to create retries directory due to collisions in '{0}' despite re-trying.</source>
        <target state="translated">Yeniden denemeye rağmen '{0} ' içindeki çakışmalar nedeniyle yeniden deneme dizini oluşturulamadı.</target>
        <note>'{0}' is the directory where collisions happen</note>
      </trans-unit>
      <trans-unit id="FailureThresholdPolicy">
        <source>Failure threshold policy is enabled, failed tests will not be restarted.</source>
        <target state="translated">Hata eşiği ilkesi etkinleştirildi, başarısız testler yeniden başlatılmaz.</target>
        <note />
      </trans-unit>
      <trans-unit id="FailureThresholdPolicyMaxCount">
        <source>Maximum failed tests threshold is {0} and {1} tests failed</source>
        <target state="translated">Maksimum başarısız test eşiği {0} ve {1} test başarısız oldu</target>
        <note />
      </trans-unit>
      <trans-unit id="FailureThresholdPolicyMaxPercentage">
        <source>Percentage failed threshold is {0}% and {1}% tests failed ({2}/{3})</source>
        <target state="translated">Başarısız olan yüzde eşiği %{0} ve başarısız %{1} ({2}/{3})</target>
        <note />
      </trans-unit>
      <trans-unit id="MoveFiles">
        <source>
=====================
Moving last attempt asset files to the default result directory
=====================
</source>
        <target state="translated">
=====================
Son deneme varlık dosyaları, varsayılan sonuç dizinine taşınıyor
=====================
</target>
        <note />
      </trans-unit>
      <trans-unit id="MovingFileToLocation">
        <source>Moving file '{0}' to '{1}'</source>
        <target state="translated">'{0}' dosyasını '{1}'e taşıma</target>
        <note />
      </trans-unit>
      <trans-unit id="RetryFailedTestsCannotStartProcessErrorMessage">
        <source>Failed to start process '{0}'</source>
        <target state="translated">'{0}' işlemi başlatılamadı</target>
        <note />
      </trans-unit>
      <trans-unit id="RetryFailedTestsExtensionDescription">
        <source>Retry failed tests feature allows to restart test execution upon failure.</source>
        <target state="translated">Başarısız testleri yeniden dene özelliği, başarısızlık durumunda test yürütmenin yeniden başlatılmasına olanak tanır.</target>
        <note />
      </trans-unit>
      <trans-unit id="RetryFailedTestsExtensionDisplayName">
        <source>Retry failed tests</source>
        <target state="translated">Başarısız testleri yeniden deneyin</target>
        <note />
      </trans-unit>
      <trans-unit id="RetryFailedTestsInvalidTestApplicationBuilderErrorMessage">
        <source>Retry failed tests only works with builders of type 'Microsoft.Testing.Platform.Builder.TestApplicationBuilder'</source>
        <target state="translated">Başarısız testleri yeniden deneme yalnızca 'Microsoft.Testing.Platform.Builder.TestApplicationBuilder' türündeki oluşturucularla çalışır</target>
        <note />
      </trans-unit>
      <trans-unit id="RetryFailedTestsMaxPercentageOptionDescription">
        <source>Disable retry mechanism if the percentage of failed tests is greater than the specified value</source>
        <target state="translated">Başarısız testlerin yüzdesi belirtilen değerden büyükse yeniden deneme mekanizmasını devre dışı bırak</target>
        <note />
      </trans-unit>
      <trans-unit id="RetryFailedTestsMaxTestsOptionDescription">
        <source>Disable retry mechanism if the number of failed tests is greater than the specified value</source>
        <target state="translated">Başarısız testlerin yüzdesi belirtilen değerden büyükse yeniden deneme mekanizmasını devre dışı bırak</target>
        <note />
      </trans-unit>
      <trans-unit id="RetryFailedTestsNotSupportedInHotReloadErrorMessage">
        <source>Retry failed tests feature is not supported in hot reload mode</source>
        <target state="translated">Başarısız testleri yeniden deneme özelliği, çalışırken yeniden yükleme modunda desteklenmez</target>
        <note />
      </trans-unit>
      <trans-unit id="RetryFailedTestsNotSupportedInServerModeErrorMessage">
        <source>Retry failed tests feature is not supported in server mode</source>
        <target state="translated">Başarısız testleri yeniden deneme özelliği sunucu modunda desteklenmiyor</target>
        <note />
      </trans-unit>
      <trans-unit id="RetryFailedTestsOptionDescription">
        <source>Enable retry failed tests</source>
        <target state="translated">Başarısız testleri yeniden denemeyi etkinleştir</target>
        <note />
      </trans-unit>
      <trans-unit id="RetryFailedTestsOptionIsMissingErrorMessage">
        <source>Option '{0}' requires option '{1}' to be specified</source>
        <target state="translated">'{0}' seçeneği, '{1}' seçeneğinin belirtilmesini gerektirir</target>
        <note />
      </trans-unit>
      <trans-unit id="RetryFailedTestsOptionSingleIntegerArgumentErrorMessage">
        <source>Option '{0}' expects a single integer argument</source>
        <target state="translated">Seçenek '{0}' tek bir tamsayı argümanı bekliyor</target>
        <note />
      </trans-unit>
      <trans-unit id="RetryFailedTestsPercentageAndCountCannotBeMixedErrorMessage">
        <source>Options '{0}' and '{1}' cannot be used together</source>
        <target state="translated">'{0}' ve '{1}' seçenekleri birlikte kullanılamaz</target>
        <note />
      </trans-unit>
      <trans-unit id="TestHostProcessExitedBeforeRetryCouldConnect">
        <source>Test host process exited before the retry service could connect to it. Exit code: {0}</source>
        <target state="translated">Yeniden deneme hizmeti ona bağlanamadan test ana makinesi işleminden çıkıldı. Çıkış kodu: {0}</target>
        <note />
      </trans-unit>
      <trans-unit id="TestSuiteCompletedSuccessfully">
        <source>
=====================
Tests suite completed successfully in {0} attempts
=====================</source>
        <target state="translated">
=====================
Test paketi {0} denemede de başarıyla tamamlandı
=====================</target>
        <note />
      </trans-unit>
      <trans-unit id="TestSuiteFailed">
        <source>
=====================
Tests suite failed, total failed tests: {0}, exit code: {1}, attempt: {2}/{3}
=====================
</source>
        <target state="translated">
=====================
Test paketi başarısız oldu, toplam başarısız test: {0}, çıkış kodu: {1}, deneme: {2}/{3}
=====================
</target>
        <note />
      </trans-unit>
      <trans-unit id="TestSuiteFailedInAllAttempts">
        <source>
=====================
Tests suite failed in all {0} attempts
=====================</source>
        <target state="translated">
=====================
Test paketi tüm {0} denemede de başarısız oldu
=====================</target>
        <note />
      </trans-unit>
      <trans-unit id="TestSuiteFailedWithWrongExitCode">
        <source>Test suite failed with and exit code different that 2 (failed tests). Failure related to an unexpected condition. Exit code '{0}'</source>
        <target state="translated">Test paketi 2 (başarısız testler) kodundan farklı bir çıkış koduyla başarısız oldu. Beklenmeyen bir koşulla ilgili hata. Çıkış kodu '{0}'</target>
        <note />
      </trans-unit>
    </body>
  </file>
</xliff>